pontoon/administration/management/commands/sync_projects.py

```diff
@@ -64,6 +65,11 @@ def handle(self, *args, **options):
         self.no_pull = options['no_pull']
 
         self.log('SYNC PROJECTS: start')
+
+        # Mark "now" at the start of sync to avoid messing with
+        # translations submitted during sync.
+        self.now = timezone.now()
```

Comment:
Shouldn't we move this into handle_project()? Storing ```self.now``` here for all projects significantly increases the time window for possible duplicate approvals we have to clean up at the end. It also makes more translations commit later than necessary.